core/src/main/resources/logback.xml

```diff
@@ -1,5 +1,9 @@
 <configuration>
 
+    <!-- Quiet down logback during startup. If you are trying to change or debug logging setup,
+         you may want to comment out this line during your development efforts. -->
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener"/>
+
```

Comment:
this is unrelated to metrics but I got too annoyed by all the logback spam when running locally so I turned it off